web/static/js/components/room.jsx

```diff
@@ -13,7 +13,9 @@ class Room extends Component {
     super(props)
```

Comment:
oh other random note (sry for spam). the community's moving away from the `jsx` file extension, in favor of good ol `.js`.